src/renderer/screens/settings/sections/Accounts/BlacklistedTokens.js

```diff
@@ -0,0 +1,132 @@
+// @flow
+
+import {
+  SettingsSection as Section,
+  SettingsSectionHeader as Header,
+} from "~/renderer/screens/settings/SettingsSection";
+import IconBan from "~/renderer/icons/Ban";
+import Text from "~/renderer/components/Text";
+import React, { useCallback, useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { findTokenById } from "@ledgerhq/live-common/lib/data/tokens";
+import CryptoCurrencyIcon from "~/renderer/components/CryptoCurrencyIcon";
+import Box from "~/renderer/components/Box";
+import styled from "styled-components";
+import IconCross from "~/renderer/icons/Cross";
+import { useDispatch, useSelector } from "react-redux";
+import { showToken } from "~/renderer/actions/settings";
+import { blacklistedTokenIdsSelector } from "~/renderer/reducers/settings";
+import { useBridgeSync } from "@ledgerhq/live-common/lib/bridge/react";
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+
+const IconContainer: ThemedComponent<{}> = styled.div`
+  color: ${p => p.theme.colors.palette.text.shade60};
+  text-align: center;
+  &:hover {
+    cursor: pointer;
+    color: ${p => p.theme.colors.palette.text.shade40};
+  }
+`;
+
+const BlacklistedTokenRow: ThemedComponent<{}> = styled(Box).attrs({
+  alignItems: "center",
+  horizontal: true,
+  flow: 1,
+  py: 1,
+})`
+  margin: 0px;
+  &:not(:last-child) {
+    border-bottom: 1px solid ${p => p.theme.colors.palette.text.shade10};
+  }
+  padding: 14px 6px;
+`;
+
+const Body: ThemedComponent<{}> = styled(Box)`
+  &:not(:empty) {
+    padding: 0 20px;
+  }
+`;
+
+const BlacklistedTokensSectionHeader: ThemedComponent<{}> = styled.div`
+  background: ${p => p.theme.colors.palette.background.default};
+  margin: 0 20px;
+  padding: 2px 12px;
+  border-radius: 4px;
+  text-transform: uppercase;
+  & > * {
+    letter-spacing: 0.1em;
+  }
+`;
+
+const BlacklistedTokens = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const sync = useBridgeSync();
+  useEffect(() => () => sync({ type: "SYNC_ALL_ACCOUNTS", priority: 5 }), [sync]);
```

Comment:
can you prefix with a code comment to explain what it does?

Reply:
Sure